invenio_stats/utils.py

```diff
@@ -46,11 +52,14 @@ def get_bucket_size(client, index, agg_field):
             }
         }
     }
-    search = Search(using=client, index=index)
+    search = Search(using=client, index=index) \
+        .filter('range', timestamp=time_range)
     search.update_from_dict(body)
```

Comment:
Instead of using `.update_from_dict(...)` in this case (which I'm not sure if it is overriding the `range` filter done above), using the `search.aggregations.metric(...)` API might be cleaner and more precise